test/scripts/e2e_subs/e2e-app-other.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+date '+keyreg-teal-test start %Y%m%d_%H%M%S'
```

Comment:
not a rekey test

Reply:
Fixed. Also fixed other tests with the same error.